src/main/java/org/launchcode/liftoffrecipetracker/controllers/CategoryController.java

```diff
@@ -49,6 +54,9 @@ public String processCreateCategoryForm(@Valid @ModelAttribute Category category
             return "category/create";
         }
 
+        User user = authenticationController.getUserFromSession(userSession);
```

Comment:
User is not set when a category is created through the copy feature

Reply:
Updated this to include ownership here